src/helper/json_helper.cpp

```diff
@@ -217,4 +217,23 @@ bool json_helper::get_json_base64_encoded_content(
 
     return true;
 }
+
+// NOLINTNEXTLINE(bugprone-easily-swappable-parameters)
+void json_helper::merge_arrays(rapidjson::Value &destination,
+    rapidjson::Value &source, rapidjson::Value::AllocatorType &allocator)
+{
+    for (auto *it = source.Begin(); it != source.End(); ++it) {
+        destination.PushBack(*it, allocator);
+    }
+}
+
+// NOLINTNEXTLINE(bugprone-easily-swappable-parameters)
+void json_helper::merge_objects(rapidjson::Value &destination,
+    rapidjson::Value &source, rapidjson::Value::AllocatorType &allocator)
+{
+    for (auto it = source.MemberBegin(); it != source.MemberEnd(); ++it) {
+        destination.AddMember(it->name, it->value, allocator);
+    }
+}
+
```

Comment:
I would suggest adding a check to verify both parameters are what it is expected. `array` or `object`

Reply:
Done